tsconfig.compileroptions.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2019",
+    "target": "es2020",
```

Comment:
This might technically be a breaking change? If so, we can leave it as is and have the *local* compiler options bump to it.

Reply:
I think we're good: https://node.green/#ES2020

It looks like ES2021 is where we start getting goodies that aren't in Node 14